_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
-| {% include anchor a="error.transport.files.maxFileSizeExceeded" %}                                          | The given file content size exceeds the max file size the backbone accepts.                                                                                                                                                                                               |
-| {% include anchor a="error.transport.files.plaintextHashMismatch" %}                                        | The actual hash of the plaintext does not match the given plaintextHash. Something went wrong while encrypting/decrypting the file.                                                                                                                                       |
-| {% include anchor a="error.transport.general.baseUrlNotSet" %}                                              | The baseUrl was not set.                                                                                                                                                                                                                                                  |
-| {% include anchor a="error.transport.general.platformClientIdNotSet" %}                                     | The platform clientId was not set.                                                                                                                                                                                                                                        |
-| {% include anchor a="error.transport.general.platformClientInvalid" %}                                      | The combination of platform clientId and clientSecret is invalid.                                                                                                                                                                                                         |
-| {% include anchor a="error.transport.general.platformClientSecretNotSet" %}                                 | The platform clientSecret was not set.                                                                                                                                                                                                                                    |
-| {% include anchor a="error.transport.identity.realmLength" %}                                               | The given realm is not 3 characters long.                                                                                                                                                                                                                                 |
```

Comment:
Removing this error message means that you have to work against release/v5 of the documentation instead of main, so it is best to adjust the base branch. (Alternatively, v5 relevant changes can be removed from this PR and added separately as a PR on v5, but I would advise against this for reasons of simplicity.)